internal/operands/metrics/reconcile_test.go

```diff
@@ -57,32 +55,16 @@ var _ = Describe("Metrics operand", func() {
 			},
 			Logger: log,
 		}
-	})
-
-	It("should create the prometheus rule", func() {
-		Expect(Reconcile(&request)).ToNot(HaveOccurred())
-		expectEqualRuleExists(newPrometheusRule(namespace), request)
-	})
-
-	It("should update the prometheus rule", func() {
```

Comment:
this test is no longer needed?

Reply:
No. The tests for the reconciliation logic were moved to `internal/common/resource_test.go`.
The tests here only check if specific resources are reconcilied, in this case the prometheus rule.